wcivf/apps/elections/templates/elections/includes/_polling_place.html

```diff
@@ -171,7 +171,7 @@ <h3>{% trans "Vote on polling day" %}</h3>
         {% if not polling_station.custom_finder and polling_station.polling_station_known and polling_station.station.geometry %}
 
             <div class="ds-card">
-                <div class="ds-card-image" id="area_map"></div>
+                <div class="ds-map-leaflet" style="aspect-ratio: 2 / 1;" id="area_map"></div>
```

Comment:
I did try this with the default `1 / 1` ratio but it just felt a little too much in this context. I knocked it back to `2 / 1` which feels more manageable and closer to what we started with.